agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -886,6 +886,14 @@ bool backhaul_manager::backhaul_fsm_main(bool &skip_select)
                 FSM_MOVE_STATE(MASTER_DISCOVERY);
             }
 
+            if (m_expected_backhaul_sta_steering.backhaul_sta_steering_enable) {
+                LOG(DEBUG) << "Sending BACKHAUL_STA_STEERING_RESPONSE_MESSAGE";
+
+                m_expected_backhaul_sta_steering.backhaul_sta_steering_enable = false;
+
+                send_cmdu_to_bus(m_expected_backhaul_sta_steering.response, controller_bridge_mac,
+                                 bridge_info.mac);
```

Comment:
Here, we should also create the response.

The bssid is active_hal->get_bssid(). Funnily enough, the station MAC address is not so easy to get: you have to get it from active_hal->get_iface_name() using get_iface_mac().

Reply:
solved